src/components/Results/ResultsRow.tsx

```diff
@@ -40,48 +38,63 @@ export const ResultsRow: FC<{result: Result}> = ({result}) => {
   const {solutions, rank_changed, rank_start, registration, subtotal, total} = result
 
   return (
-    <div className={styles.rowWrapper}>
-      <div className={styles.rank}>
+    // `display: 'contents'` je taka HTML verzia fragmentu - tento div sa v strukture nepocita pre ucely parent gridu,
+    // miesto toho sa pouzivaju jednotlive children. preto aj hover tu targetuje children divy, lebo parent v podstate nema velkost
+    // takisto `justifyContent: 'center'` je na vsetkych elementoch osobitne
+    <Box sx={{display: 'contents', ':hover > div': {bgcolor: 'black', color: 'white'}}}>
+      <Stack
+        sx={{
+          justifyContent: 'center',
+          // top-level PageLayout padding je vypnuty, odsadzame to rucne tu
+          pl: 1,
+        }}
+      >
         <Typography variant="h1" component="span">
           {rank_changed && rank_start + '.'}
         </Typography>
-      </div>
-      <div className={styles.nameAndSchool}>
-        <Typography variant="h2" component="span" className={styles.name}>
+      </Stack>
+      <Stack sx={{px: '10px', justifyContent: 'center'}}>
+        <Typography variant="h3" component="span">
           {registration.profile.first_name + ' ' + registration.profile.last_name}
         </Typography>
-        <Typography variant="body3" className={styles.school}>
+        <Typography variant="body3">
           {registration.school.name + ' ' + registration.school.street + ' ' + registration.school.city}
         </Typography>
-      </div>
-      <div className={styles.grade}>
+      </Stack>
+      <Stack sx={{justifyContent: 'center'}}>
         <Typography variant="h3" component="span" fontWeight={400} fontStyle="italic">
           {registration.grade}
         </Typography>
-      </div>
-      <div className={styles.score}>
+      </Stack>
+      <Stack sx={{justifyContent: 'center', px: '10px'}}>
         {solutions.map((series, index) => (
-          <div key={index}>
+          <Stack key={index} direction="row">
             {series.map((solution, index) => (
-              <Typography variant="body2" key={index}>
+              <Typography key={index} variant="body2" sx={{width: '18px', textAlign: 'center'}}>
                 {solution.points}
               </Typography>
             ))}
             <Tooltip title={`Súčet bodov za ${index + 1}. sériu po uplatnení bonifikácie`}>
-              <Typography variant="body2" fontWeight={600} className={styles.subtotal}>
+              <Typography variant="body2" fontWeight={600} sx={{width: '25px', textAlign: 'center'}}>
```

Comment:
paci sa mi ako si nahradil padding fixnou sirkou

Reply:
toto tam actually bolo v tom grid template ako 6x 25px + 1x 18px

Reply:
a aj napriek tomu mal ten text paddingy? (porovnaval som to ale asi mi to uslo)

Reply:
noo, k tomu tam bolo este justifyItems: center na parentovi, co teda nejako v tom layoute fungovalo, ale radsej som to tu prepisal na textAlign, takze az to tomu z toho robi to iste co predtym